### ISSUE TITLE
Can't place landmarks in latest (2018/05/26) release

### ISSUE THREAD

**Issue by david-hoffman** [saalfeldlab/bigwarp#44]
Looks like space bar doesn't do anything in the latest release, i.e. I can't place or edit landmarks. This is on a windows machine.

**Comment by david-hoffman** [comment]
@ctrueden I tried your first two suggestions, but they didn't seem to change anything. I downgraded to https://downloads.imagej.net/fiji/Life-Line/fiji-win64-20170530.zip which fixed the problem.

I spoke with @bogovicj and he's looking into it currently.

**Comment by bogovicj** [comment]
I can reproduce this issue.
Thanks @david-hoffman

**Comment by david-hoffman** [comment]
@bogovicj I take it back, downgrading may have fixed the space bar issue, but it introduced a new issue for large data:

```java
(Fiji Is Just) ImageJ 2.0.0-rc-59/1.51n; Java 1.8.0_66 [64-bit]; Windows 10 10.0; 62543MB of 785287MB (7%)
 
java.lang.ArrayIndexOutOfBoundsException: 2
	at bigwarp.BigWarpInit.wrapSourcesAsRenamable(BigWarpInit.java:276)
	at bigwarp.BigWarpInit.createBigWarpData(BigWarpInit.java:257)
	at bigwarp.BigWarpInit.createBigWarpData(BigWarpInit.java:346)
	at bigwarp.BigWarpInit.createBigWarpDataFromImages(BigWarpInit.java:408)
	at bdv.ij.BigWarpImagePlusPlugIn.run(BigWarpImagePlusPlugIn.java:79)
	at ij.IJ.runUserPlugIn(IJ.java:217)
	at ij.IJ.runPlugIn(IJ.java:181)
	at ij.Executer.runCommand(Executer.java:137)
	at ij.Executer.run(Executer.java:66)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by bogovicj** [comment]
Yikes! thanks for heads up.

**Comment by bogovicj** [comment]
Linking my related question on the imagej forum:
http://forum.imagej.net/t/ui-behavior-not-distinguishing-pressed-released/11189

**Comment by bogovicj** [comment]
The next bigwarp release will use this and change the behavior of "space"

**Comment by mtakeno** [comment]
I was able to reproduce the original issue (Can't place landmarks) on the latest Win64 release (downloaded 2018-09-18). Was able to downgrade as mentioned above to the lifeline 20170530 release and recover landmark behavior with the space bar to align 949 MB and 705 MB datasets, and 1.7 GB and 705 MB datasets without an OutOfBoundsException.

**Comment by bogovicj** [comment]
Release coming soon, there's an outstanding bug to fix beforehand (and I don't have tons of time atm).

In case you're interested @mtakeno, an alternative to downgrading is to install the latest SNAPSHOT release into Fiji ([see here](https://forum.image.sc/t/how-to-align-images-manually-and-merge-them-into-a-multi-channel-composite-image/11071/5?u=bogovicj))

**Comment by mtakeno** [comment]
Fantastic, it appears to be fixed with the daily snapshot 3.1.2 release. Thank you for your prompt response! I'm signing up on image.sc to follow. Bigwarp has been an immensely important tool in our work, and will certainly be credited upon publication.
